frontend/ui/lib/services/places_services_html.dart

```diff
@@ -0,0 +1,64 @@
+import 'dart:async';
+import 'dart:html';
+import 'package:flutter/material.dart';
+import 'package:google_maps/google_maps.dart';
+import 'package:google_maps/google_maps_places.dart';
+import 'package:tripmeout/model/place_visit.dart';
+
+class PlacesApiServices {
+  final AutocompleteService autocompleteService = AutocompleteService();
+  final PlacesService placesService =
+      PlacesService(document.getElementById("maps"));
+
+  Future<List<PlaceVisit>> getAutocomplete(
```

Comment:
This shouldn't be a PlaceVisit.

so a `PlaceVisit` is a visit to a place in the context of a `Trip` where both `PlaceVisit` and `Trip` are model objects in OUR system. There is the separate idea of a `PlacesApiPlace` which is referenced by both `Trip` and `PlaceVisit` but exists in an external system. `PlaceVisit` != `PlacesApiPlace`

The `PlacesApiService` should deal with `PlacesApiPlace` or whatever you want to call it, not `PlaceVisit`s.

Reply:
This is a larger refactoring problem, not a small one, so I'm only leaving the one comment because I don't want to review the rest of this till this is done.